### DISCUSSION TITLE
Rotary switch change other button's CC

### DISCUSSION THREAD

**Discussion by qchaillet** [shanteacontrols/OpenDeck#112] [General]
First of all : amazing job ! 👍 👍 

I am wondering if I could use a rotary switch? ([like this one ?](https://proto-pic.co.uk/product/sparkfun-com-13253-rotary-switch-10-position/))
Based on its current position, others buttons / rotary encoders / led may have a different CC assigned.

Ex : rotary switch in position 1 : encoder3 is set to update the CC32 value, encoder5 is on CC48
rotary switch in position 2 : encoder3 is set to update the CC20 value, encoder5 is on CC19

I guess the rotary switch can send different sysex information based on his position, but I m not sure ...

Thank you :-)

**Comment by paradajz** [comment]
Thanks!

Yes, you can use that. Rotary switch is basically multiple switches, so just connect it as you would normal button.
You cannot, however, use it to change the values of other components - you can use it to switch presets, which would achieve similar effect.

**Reply by qchaillet** [reply]
Hope I can save a good number of preset so, and that I can duplicate them easily ( I do no have the hardware yet to test those features).
Thanks for the answer :-)
